lib/internal/bootstrap_node.js

```diff
@@ -283,6 +282,12 @@
       if (!caught)
         caught = process.emit('uncaughtException', er);
 
+      // After this call is made the stack is gone.
+      // TODO(trevnorris): Do we want to run all after callbacks here to let
+      // them know an error has occurred? Since it may be a little confusing
+      // that the after callbacks were never called.
```

Comment:
Maybe we'd need a `doCallbackWithAfter(id, cb, ...args)` so we can guarentee it? Unsure.